README

```diff
@@ -1,14 +1,25 @@
-Usage
-1. Install the rsv-vo-gwms RPM
-2. Disable gratia in /etc/osg/config.d/30-rsv.ini
-3. Enable the gfactory vo query
-rsv-control --enable org.osg.local-gfactory-querying-local --host myhost
-4. Modify /etc/rsv/rsv.conf
-And update proxy-file entry to your VO proxy.
+#rsv-vo-gwms
+
+Architecure
+===========
+rsv-vo-gwms architecture is identical to the one of [rvs-gwms-tester](https://github.com/opensciencegrid/rsv-gwms-tester).
+Except it submits Condor-C jobs instead of Condor Vanilla jobs. In this regard it acts as a [GlideinWMS Factory](https://glideinwms.fnal.gov/doc.prd/factory/index.html). This way it does not need a GlideinWMS Frontend to work as rsv-vo-gwms does.
+
+Requirements
+============
 
+1. An X509 certificate with the VOMS extension of the Virtual Organization (VO).
+1. A RedHat running machine where rsv-gwms-tester can be installed as root
 
-Where myhost is the host where rsv-vo-gwms is installed
+Brief Installation
+===================
+
+1. Install the rsv-vo-gwms RPM. `yum install rsv-vo-gwms`
```

Comment:
Doesn't this need the devops or contrib repo enabled?

Reply:
Actually this package is not longer in any repository. 

```console

Available 
rsv.noarch 
rsv-consumers.noarch 
rsv-consumers-zabbix.noarch 
rsv-core.noarch 
rsv-gwms-tester.noarch 
rsv-metrics.noarch 
rsv-perfsonar.noarch 
```
So maybe we can just change the README to this package is not supported neither carried anymore?

@brianhlin

Reply:
Works for me